### PR TITLE
Update 01.abstract.md

### PR DESCRIPTION
**Pull request by Farnazmdi**

### CODE REVIEW

**Comment by Farnazmdi** [comment]
@aarmey in the first paragraph of figure 5 in the results, we have this sentence: "Cells of even very distinct molecular states can have partly overlapping phenotypes `due to non-heritable variation`. Therefore, we sought to evaluate how different two states would need to be for us to accurately identify them as distinct."
I feel the first sentence is a bit confusing but I am not sure how to edit it so I won't state something wrong. Do you find it confusing too?

**Comment by aarmey** [comment]
Indeed—I also think it could be confusing, but couldn't come up with another accurate way to describe the point.

**Comment by Farnazmdi** [comment]
I think these changes cover the comments. Some of the comments were already applied and I had forgotten to reply to them.
I added one reference which I am not 100% sure that it is suitable for what you intended. I appreciate it if you could check and see if it matches. It is in the introduction and the paper is:
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0223183

File: manuscript/01.introduction.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ Large-scale profiling studies can find molecular features that associate with dr
...
...asurement provides accumulating evidence for a prediction [@DOI:10.1126/scisignal.aao1917]. HMMs have been adapted to lineage trees (tHMMs) so that each measurement across the tree can similarly provide accumulating evidence for a prediction. Just like with time-series data, these models can provide very accurate predictions despite noisy measurements and limited information by recognizing the co-dependence between related measurements [@doi:10.1109/78.668544; @doi:10.1109/TSP.2004.832006]. tHMMs have been used in a multitude of applications, from image classification to comparative genomics [@pmid:18255546; @pmid:23762278]. In cells, these models have been fit to lineages collected from stem cells and bacteria colonies but have always required tailor-made implementations [@PMID:19628503; @doi:10.1101/488981]. Improvements in cell tracking and high-throughput imaging promise to make these models valuable techniques for studying the plasticity of heterogeneous cell populations. However, widespread use will require usable implementations that can be readily adapted to different experimental measurements, examples of unique insights revealed, and standards for experimental validation.
```
Comment (aarmey) [inline manuscript/01.introduction.md:14]: Why this paper?

File: manuscript/01.introduction.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ Large-scale profiling studies can find molecular features that associate with dr
...
 
 <!-- Introduction to the paper -->
 
-Here, we develop a generic implementation of tHMMs with a defined interface for integrating diverse types of measurements on cell lineage trees. We use this to analyze how populations of breast cancer cells respond to therapy with a cell cycle reporter. Single cell measurements of the cell cycle revealed extensive variation in drug response that is not captured in population-level measurement. This model allows us to quantify the dynamics and phenotypic features of heterogeneity in drug response. We are able to predict the most likely number of phenotypically distinct subpopulations, the characteristics of those subpopulations, transition probabilities from one state to another, and each cell's expected state. This work, therefore, provides a flexible phenotype-driven route to discovering cell-to-cell variation in drug response, demonstrates an overall strategy for quantifying the dynamics of cell heterogeneity, and implements a very general software tool for widespread use of tHMM models.
```
Comment (aarmey) [inline manuscript/01.introduction.md:18]: It's not just parent-child relationships, though...

File: manuscript/03.results.md [hunk 3]
```diff
@@ -4,16 +4,15 @@
...
...neage). Gemcitabine is a chemotherapy agent that disrupts DNA replication and results in extension of and apoptosis in S phase [@pmid:11504793]. Comparing a few random lineages in control condition and gemcitabine-treated cells demonstrates that while there is a striking difference in the growth dynamics of individual cells, the total number of cells at any given time is nearly identical in these two conditions. As a result, population-level drug response measurements cannot distinguish the differences in these two conditions. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to distinguish these two populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich measurements to quantify the specific structure of heterogeneous response. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel [@doi:10.1126/science.abc1944].
```
Comment (aarmey) [inline manuscript/03.results.md:7]: Does this paper reference demonstrate the many different measurements one can make in single cells? Why reference it here?

File: manuscript/03.results.md [hunk 4]
```diff
@@ -4,16 +4,15 @@
...
...trees can provide, we wished to implement a strategy for classifying cells based on their phenotype and relationships. Given a set of measurements made across a lineage tree, we used a lineage tree-based hidden Markov model (tHMM) to fit these data (Fig. {@fig:tree}a). Like a typical hidden Markov model, a tHMM can infer the hidden discrete "states" of cells given a series of measurements where a state could be defined by specific phenotype distributions. In this study, cells that belong to a "state" are a subpopulation of cells that share a common trait; for instance, cells that are highly arrested in their G1 phase with high concentrations of a drug, could belong to one state. Of note, these states are only identified based on measurable observations from an experiment. The inference of these states takes place using an iterative strategy wherein the states of each cell are predicted given the phenotype of not only the cell but also all other cells in their lineage (“expectation” step), and then each distribution of phenotypes is fit to match the cells within that state (“maximization” step) (Fig. {@fig:tree}b). This expectation-maximization (EM) process repeats until convergenc.
```
Comment (aarmey) [inline manuscript/03.results.md:15]: Do you mean for the two paragraphs to be merged here? There must be a blank line between paragraphs for them to remain separate.

**Review by aarmey — CHANGES_REQUESTED** [review]
A few questions.

File: manuscript/01.introduction.md (hunk 1 above)
Comment (Farnazmdi) [reply manuscript/01.introduction.md:14]: It talks about the application of HMM for time series data... Isn't it the point?

File: manuscript/01.introduction.md (hunk 2 above)
Comment (Farnazmdi) [reply manuscript/01.introduction.md:18]: I agree but thought basically other relationships stem from that.. I meant we don't exactly focus on say sister-sister relationship. Should I explain more in that paragraph?

File: manuscript/03.results.md (hunk 4 above)
Comment (Farnazmdi) [reply manuscript/03.results.md]: I didn't mean to do that. I just fixed it. Also fixed the misspelling of the last word.

File: manuscript/03.results.md (hunk 3 above)
Comment (Farnazmdi) [reply manuscript/03.results.md:7]: It doesn't. I should have placed it somewhere else... I will replace it with another reference.

File: manuscript/01.introduction.md (hunk 1 above)
Comment (Farnazmdi) [reply manuscript/01.introduction.md:14]: Should I replace this?

Comment (aarmey) [reply manuscript/01.introduction.md:14]: This is a poor example. Surely there is a more expansive paper or review that covers this topic in general, like this sentence is meant to address.

Comment (Farnazmdi) [reply manuscript/01.introduction.md]: I have a couple of papers open in my browser for 3 days and I have not been able to finish reading them. I returned this to `(CITE)` so I can later choose the best match literature to it.

File: manuscript/01.introduction.md (hunk 2 above)
Comment (Farnazmdi) [reply manuscript/01.introduction.md:18]: This sentence is modified to "We use the relationship between the cells (that share the same ancestor or belong to the same lineage tree) to analyze how populations of breast cancer cells respond to therapy with a cell cycle reporter."

**Comment by Farnazmdi** [comment]
@aarmey please let me know if this is ready to merge.
I will work on references on another branch as it is taking me longer than expected.

File: manuscript/03.results.md [hunk 5]
```diff
@@ -4,15 +4,15 @@
...
...on between the measured observations, for instance the duration of G1 phase is very close between sister cells, especially in control condition (Fig. @fig:lineage a), and the duration of each cell's phase is reflected through its dauther cell. These variation gets more complicated as we treat cells with different compounds (Fig @fig:lineage b). As a result, population-level drug response measurements provides an average of dynamical behaviors that mask variations existing in a population of cells over time. Measurements that record the history of cells (e.g., CFSE staining, Luria-Delbruck experiment) can help to identify the various dynamics within cell populations but must make assumptions about the dynamics of heterogeneity [@DOI:10.1073/pnas.1715639115; @PMID:28607484]. Lineage measurements, by contrast, provide sufficiently rich measurements to quantify the specific structure of heterogeneous response. By observing individual cells, a variety of phenotypic measurements such as inter-mitotic times, cell cycle phase durations, motility, cell death and division, morphology, and protein and transcription factor levels can be characterized in parallel [@doi:10.1126/science.abc1944].
```
Comment (aarmey) [inline manuscript/03.results.md:7]: Please put this change on a separate branch.

File: manuscript/03.results.md [hunk 6]
```diff
@@ -58,18 +58,18 @@ One does not usually know the number of distinct cell states within a population
...
...ell cycle arrest in G1 phase ([@doi:10.4161/cc.25728]). AU565 cells were treated with 25 nM, 50 nM, and 250 nM of lapatinib during the experiment and imaged and tracked for over 96 hours. We fitted the lapatinib treated data to the model with 6 states, based on our BIC-based model selection (Fig. {@fig:expBIC}a). As a result of fitting, the population was classified into stable states and cycles (Fig. {@fig:emissionsLPT}a). Lineages that belong to the same state showed that increasing the lapatinib concentration results in the increased mean of G1 lengths in almost all states, especially in cells of state 1 and 4 (Figs. {@fig:emissionsLPT}b-f, @fig:LapatinibTree) and almost no change in the mean of G2 lengths (Fig. {@fig:emissionsLPT}g). The probability of cell death at G1 phase was higher in state 2 and 3, and in G2 phase only cells in state 3 partially die. The cell death effects appear only when the concentration of the drug is more than 50 nM (Figs. {@fig:emissionsLPT}h-i). Further, cells in states other than 2 and 3 seem to be resistant to cell death in all concentrations, and the effect of the drug in these cells is solely on G1 phase extension (Figs. {@fig:emissionsLPT}f-i).
```
Comment (aarmey) [inline manuscript/03.results.md:67]: Remove the citation.

**Review by aarmey — CHANGES_REQUESTED** [review]
A couple small changes before merging.

File: manuscript/03.results.md (hunk 5 above)
Comment (Farnazmdi) [reply manuscript/03.results.md]: done.

File: manuscript/03.results.md (hunk 6 above)
Comment (Farnazmdi) [reply manuscript/03.results.md]: done.

**Review by aarmey — APPROVED** [review]
Great! This is ready.

**Pull request merged**